src/components/AddNote.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useContext } from 'react';
+import styled from 'styled-components';
+import { Icon } from 'semantic-ui-react';
+import { mainStateDispatchContext } from './Main';
+
+const AddNoteWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: black;
+`;
+
+const AddNoteButton = styled.button`
+  background: none;
+  border: none;
+  outline: none;
+  font-size: 3rem;
+  color:inherit;
+`;
+
+function AddNote() {
+  /**
+   * the dispatch function of the state of 'Main' component
+   * @param {object} action
+   */
+  const mainStateDispatchFunction = useContext(mainStateDispatchContext);
+
+  const [noteTemp, setNoteTemp] = useState({
+    id: '',
+    body: '',
+    color: '#FEF3BD',
+    timeAdded: '',
+  });
+
+  const addIcon = <Icon name="add" />;
```

Comment:
Consistency alert!
Use '' instead of double quote.

Reply:
Done